### ISSUE TITLE
Document how to handle custom fonts.

### ISSUE THREAD

**Issue by swistakm** [swistakm/pyimgui#34]

**Comment by iliyang** [comment]
Does this mean it's currently possible and is only undocumented? If yes, would it be easy to guess by looking at the API and in the original imgui code/examples?

**Comment by iliyang** [comment]
I just tried to figure it on my own and get an assertion error in imgui. What I do is the following:

```python
font = imgui.get_io().fonts.add_font_from_file_ttf("fonts/DroidSans.ttf", 16)
...
while not glfw.window_should_close(self.window):
    # draw stuff
    imgui.push_font(font)
    # draw text
    imgui.pop_font()
```
The font loads successfully, i.e. the `font` variable is a valid object, not `None`. But then I get an error where `push_font` is called:
```
Traceback (most recent call last):
  File "/Users/...[location].../igui.py", line 200, in <module>
    app.main_loop()
  File "/Users/...[location].../igui.py", line 76, in main_loop
    self.draw()
  File "/Users/...[location].../igui.py", line 86, in draw
    self.draw_contents()
  File "/Users...[location].../igui.py", line 152, in draw_contents
    imgui.push_font(font)
  File "imgui/core.pyx", line 4757, in imgui.core.push_font
imgui.core.ImGuiError: ImGui assertion error (font && font->IsLoaded()) at imgui-cpp/imgui.cpp:4529
```
Any ideas will be greatly appreciated! I'm running macOS 10.12, tried the above code with both Python 2.7 and 3.6, both fail the same way.

**Comment by iliyang** [comment]
@swistakm was kind to explain how this works in #50.

**Comment by swistakm** [comment]
I have finally documented it in #110

**Comment by swistakm** [comment]
Released as `1.0.0`
